Hive-Five-project/hive-five
# [FRONT] Add apiary, beehive home and generic UI

## Features
- Apiary page: Show details about the apiary and the list of beehives of this apiary
- Beehive page: Show details about the beehive
- Added translations for apiary form (It was hard coded)
- Added a generic kebab menu with a menu when you click on it (Can change icons, path, color and text)
- "ListCard" component is a bit more generic now, the add card css is no longer hard coded in each list (just check the "colorInverted" props)

## How to test
1. Go to http://127.0.0.1:63281/apiary/list
2. Click on an Apiary to be redirected to the Apiary details information page, that shows details and the beehive list (The "adding a beehive" button is a placeholder and is not working):

![image](https://github.com/Hive-Five-project/hive-five/assets/55053185/5b97a411-f7f7-42d6-a83b-264671057cf8)

3. Click on the kebab menu at the top right to see the edit and delete button (The edit button should redirect you to the apiary edit page and the delete button is a placeholder and not yet implemented)

![image](https://github.com/Hive-Five-project/hive-five/assets/55053185/4eff4ba1-bc45-4e73-b7d1-50c19509fba7)

4. Click on a beehive to be redirected to the beehive details information page:

![image](https://github.com/Hive-Five-project/hive-five/assets/55053185/28903abb-34a4-455d-bcab-3e0a6929a77e)

5. You can click on the kebab menu but both buttons are not yet implemented:

![image](https://github.com/Hive-Five-project/hive-five/assets/55053185/190d8938-21fa-4abc-997f-8e3e42d079ed)
